include/CoreGraphics/CGFont.h

```diff
@@ -35,11 +37,11 @@ COREGRAPHICS_EXPORT const CFStringRef kCGFontVariationAxisDefaultValue;
 
 COREGRAPHICS_EXPORT void CGFontRelease(CGFontRef font);
 COREGRAPHICS_EXPORT CGFontRef CGFontRetain(CGFontRef font);
-COREGRAPHICS_EXPORT CGFontRef CGFontCreateWithDataProvider(CGDataProviderRef provider);
+COREGRAPHICS_EXPORT CGFontRef CGFontCreateWithDataProvider(CGDataProviderRef provider) STUB_METHOD;
```

Comment:
wrote this before we re-evaluated the line, this is pretty non-blocking, I'm going to file a separate issue for this.

---
In reply to: [83305866](https://github.com/Microsoft/WinObjC/pull/1145#discussion_r83305866) [](ancestors = 83305866)